Simplified/en.lproj/Localizable.strings

```diff
@@ -270,3 +270,9 @@
 
 // Announcement
 "Announcement" = "Announcement";
+
+// Library Finder
```

Comment:
here's the Italian strings:
```
"Find a Library" = "Trova una biblioteca";
"Failed to update library list" = "Si è verificato durante l'aggiornamento della lista delle biblioteche";
"Location, library name or zip code" = "Luogo, nome della biblioteca o CAP";
"You can search for your library by name, branch location, or your own location" = "Puoi cercare la tua biblioteca per nome o luogo"
```

Reply:
Thanks, do you mind translating `My Libraries` and `Add New Library` as well?

Reply:
oops I missed them:
"My Libraries" = "Le mie biblioteche";
 "Add New Library" = "Aggiungi nuova biblioteca"